package.json

```diff
@@ -1,55 +1,27 @@
 {
-  "name": "ally",
+  "name": "tokenator",
   "version": "1.0.0",
-  "description": "Ally token contract",
-  "main": "index.js",
-  "scripts": {
-    "compile": "waffle",
-    "deploy:mainnet": "hardhat deploy --network mainnet --tags ethereum",
-    "verify:mainnet": "hardhat etherscan-verify --network mainnet",
-    "deploy:kovan": "hardhat deploy --network kovan --tags kovan",
-    "verify:kovan": "hardhat etherscan-verify --network kovan",
-    "test": "mocha",
-    "test:main": "bash ./scripts/test-main.sh",
-    "test:swap": "bash ./scripts/test-swap.sh"
+  "description": "Tokenator contract",
+  "directories": {
+    "test": "test"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ichifarm/ally.git"
+  "scripts": {
```

Comment:
why so minimal? what about hardhat compile, hardhat test, linting, formatting scripts?

Given that this is a hardhat only project(since it doesn't use forge) I'd recommend copying over the hardhat project opinionated set up from this template:

https://github.com/PaulRBerg/hardhat-template

Reply:
added scripts for compile and test